pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages (eg. python=2.7):
```

Comment:
There certainly are, but I can't give you a concrete example from the top of my head, I'd have to look at a few pages looking for that. In any case, I don't see many ways this could be written: just use two common packages as you would use in a real command, and wrap them in the token markers. In the description, you could say something like "installing two named packages into it".